suite-common/token-definitions/src/tokenDefinitionsSelectors.ts

```diff
@@ -23,10 +23,13 @@ export const selectNftDefinitions = (
 ) => state.tokenDefinitions?.[networkSymbol]?.nft;
 
 export const selectCoinDefinition = (
-    state: TokenDefinitionsRootState,
+    stateTracked: TokenDefinitionsRootState,
     networkSymbol: NetworkSymbol,
     contractAddress: TokenAddress,
 ) => {
+    // We need to use getUntracked here otherwise this could interfere with memoization in isTokenDefinitionKnown
+    const state = getUntracked(stateTracked) ?? stateTracked;
```

Comment:
In docs they say that: ```[Notes] This function is for debugging purpose. It’s not supposed to be used in production and it’s subject to change.```

might that cause any problem?

Reply:
I don't think so, since we are not upgrading that lib anyway. Also this sadly don't have any other reasonable solution